docs/Configuration.md

```diff
@@ -302,6 +302,66 @@ shims:
         legacy-library-a: src/library.a
         legacy-library-b: src/library.b
 ```
+## Custom Bundling
+
+Custom bundles can be defined in the `ui5.yaml`. It should be located in the `builder` configuration. With the property `bundles` a list of `bundleDefinitions` can be described.
+
+```yaml
+builder:
+  bundles:
+    - bundleDefinition:
+        name: "sap-ui-custom.js"
+        defaultFileTypes:
+          - ".js"
+        sections:
+          - mode: raw
+            filters:
+            - ui5loader-autoconfig.js
+            resolve: true
+            sort: true
+        bundleOptions:
```

Comment:
Shouldn't this be on the same indentation-level as the `bundleDefinition`?

Reply:
Good eyes! :D Done!